exporting/opentsdb/README.md

```diff
@@ -0,0 +1,38 @@
+<!--
+title: "Export metrics to OpenTSDB with HTTP"
+description: "Archive your Agent's metrics to a OpenTSDB database for long-term storage and further analysis."
+custom_edit_url: https://github.com/netdata/netdata/edit/master/exporting/opentsdb/README.md
+sidebar_label: OpenTSDB with HTTP
+-->
+
+# Export metrics to OpenTSDB with HTTP
+
+Netdata can easily communicate with OpenTSDB using HTTP API. To enable this channel, run `./edit-config exporting.conf`
+in the Netdata configuration directory and set the following options:
+
+```conf
+[opentsdb:http:my_instance]
```

Comment:
We are missing a `enabled = yes`, here, because the default value for every instance is `0'.

Reply:
Done